salt/modules/virt.py

```diff
@@ -1747,6 +1779,7 @@ def init(
     boot_dev=None,
     numatune=None,
     iothreads=None,
+    cputune=None,
```

Comment:
since this is adding tuning of the virtual cpu, I would move it in the `cpu` param, maybe a `tuning` key of it.